Support/Gemfile

```diff
@@ -1,3 +1,3 @@
 source "http://rubygems.org"
 
-gem "rspec", '~> 3.3'
+gem "rspec", ['~> 3.3', '< 3.5']
```

Comment:
Hm, why is this necessary? `TextMateFormatter` is now part of this bundle (see https://github.com/rspec/rspec.tmbundle/blob/master/Support/lib/rspec/mate/text_mate_formatter.rb), so the bundle should work with all RSpec versions.